api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -117,6 +117,13 @@ message JwtProvider {
     //      inline_string: ACADADADADA
     //
     config.core.v3.DataSource local_jwks = 4;
+
+
+    // The two below fields define the amount of slack in seconds that will be used
+    // when determining if a JWT is valid or has expired. Validity is determined by
+    // the formula: VALID_FROM("iat") - nbf_slack_slack <= NOW <= VALID_TO("exp") + exp_slack.    
+    int64 nbf_slack = 0;
```

Comment:
we can use int32.   Please check out this protobuf [guide](https://developers.google.com/protocol-buffers/docs/overview)

default value for int32 is 0 in protobuf, not need to specify it

Reply:
Actually, in Envoy we override Google style and prefer `uint32` :)

Reply:
Unless there are absolute high-performance needs (which we don't have here) you should always make values explicit. This is clear to anyone without further poking any documentation.